landingzones/tests/test_permissions.py

```diff
@@ -200,6 +200,95 @@ def test_zone_create_disable(self):
         self.assert_response(url, good_users, 200)
         self.assert_response(url, bad_users, 302)
 
+    def test_zone_update(self):
+        """Test ZoneUpdateView permissions"""
+        url = reverse(
+            'landingzones:update',
+            kwargs={'landingzone': self.landing_zone.sodar_uuid},
+        )
+        good_users = [
+            self.superuser,
+            self.user_owner_cat,
+            self.user_owner,
+            self.user_delegate_cat,
+            self.user_delegate,
+            self.user_contributor,
+        ]
+        bad_users = [
+            self.user_contributor_cat,
+            self.user_guest_cat,
+            self.user_guest,
+            self.user_no_roles,
+            self.user_finder_cat,
```

Comment:
For consistency and easier verification & modification of tests, please place users in permission tests in the canonical order: superuser -> inherited category users (by rank) -> local project users (by rank) -> users without roles.

The complete ordering is as follows:

```
self.superuser,
self.user_owner_cat,
self.user_delegate_cat,
self.user_contributor_cat,
self.user_guest_cat,
self.user_finder_cat,
self.user_owner,
self.user_delegate,
self.user_contributor,
self.user_guest,
self.user_no_roles,
self.anonymous,
```